drgn_tools/block.py

```diff
@@ -512,6 +512,21 @@ def get_inflight_io_nr(prog: drgn.Program, disk: Object) -> int:
     return nr
 
 
+def print_block_devs_info(prog: drgn.Program) -> None:
+    """
+    Prints the block device information
+    """
+    output = [["MAJOR", "GENDISK", "NAME", "REQUEST_QUEUE", "Inflight I/Os"]]
+    for disk in for_each_disk(prog):
+        mjr = int(disk.major)
+        gendisk = hex(disk.value_())
+        name = disk.disk_name.string_().decode("utf-8")
+        rq = hex(disk.queue.value_())
+        ios = int(get_inflight_io_nr(prog, disk))
+        output.append([str(mjr), gendisk, name, rq, str(ios)])
```

Comment:
Do we really need converting 'mjr' and 'ios'?

Reply:
yes, pre_commit complained that the list has different types of objects.

I have use int conversion for major because it has "(int 32)" prefix, and for IOs as you mentioned it does not need int conversion, i will remove that. Converted both of them to str to make pre_commit happy.